applications/osb-portal/package.json

```diff
@@ -19,41 +19,43 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.10.1",
```

Comment:
why is material ui a dev dependency?

Reply:
same for react, redux, etc

Reply:
because they are loaded as js in the index.html

Reply:
now I remember, thanks